tools/PostProcessing/stm_module.f90

```diff
@@ -348,16 +348,21 @@ subroutine tersoff_hamann
     call set_prefac_real(9)
     allocate(current(nptsx,nptsy,nptsz))
     allocate(psi(nptsx,nptsy,nptsz))
+
+    if (nspin==1 .and. fermi_offset.ne.zero) &
```

Comment:
Maybe remove `nspin==1` here

Reply:
Thank you for your comment.
I removed nspin==1
and also changed the digit of the output from f7.3 to f10.3.